pkg/test/framework/examples/example.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package examples
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+type outputType string
+
+const (
+	//TextOutput describes a test which returns text output
+	TextOutput outputType = "text"
+	//YamlOutput describes a test which returns yaml output
+	YamlOutput outputType = "yaml"
+	//JSONOutput describes a test which returns json output
+	JSONOutput outputType = "json"
+)
+
+const (
+	istioPath = "istio.io/istio/"
+)
+
+//CreateExample returns an instance of an example test
+func CreateExample(t *testing.T, name string) Example {
```

Comment:
By  convention in Go, this should either be NewExample or simply New, leading to either of the following at  the call site:

ex := example.NewExample(...)   // a bit of studering
ex := example.New(...)  // no studering.

Reply:
Fixed